javascript/react/src/components/Codebook/CodebookFieldEditor.stories.tsx

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { CodebookSchema, Field } from '@datashaper/schema'
+import type { ComponentStory } from '@storybook/react'
+import { useState } from 'react'
+
+import { CodebookFieldEditor } from './CodebookFieldEditor.js'
+import {
+	CodebookFields,
+	DEFAULT_CODEBOOK_FIELDS,
+} from './CodebookFieldEditor.types.js'
+
+const storyMetadata = {
+	title: 'Components/CodebookFieldEditor',
+	component: CodebookFieldEditor,
+	argTypes: {
+		tableView: {
+			control: 'boolean',
+		},
+		hideLabel: {
+			control: 'boolean',
+		},
+		visibleFields: {
+			control: 'inline-check',
+			options: [
+				CodebookFields.DisplayName,
+				CodebookFields.Description,
+				CodebookFields.DataType,
+				CodebookFields.DataNature,
+				CodebookFields.Units,
+				CodebookFields.Mapping,
+			],
+		},
+	},
+}
+export default storyMetadata
+
+const codebookResult = {
+	$schema: 'http://json-schema.org/draft-07/schema#',
+	id: 'http://json-schema.org/draft-07/schema#',
+	name: 'Generator',
+	fields: [
+		{
+			name: 'ID',
+			type: 'number',
+			nature: 'ordinal',
+			metadata: {
+				bins: [
+					{
+						min: 5.87,
+						count: 759,
+					},
+					{
+						min: 358.428,
+						count: 0,
+					},
+					{
+						min: 710.986,
+						count: 1,
+					},
+					{
+						min: 1063.5439999999999,
+						count: 74,
+					},
+					{
+						min: 1416.1019999999999,
+						count: 164,
+					},
+					{
+						min: 1768.6599999999999,
+						count: 75,
+					},
+					{
+						min: 2121.218,
+						count: 43,
+					},
+					{
+						min: 2473.776,
+						count: 21,
+					},
+					{
+						min: 2826.334,
+						count: 72,
+					},
+					{
+						min: 3178.892,
+						count: 56,
+					},
+				],
+				count: 1265,
+				distinct: 1182,
+				maximum: 3531.45,
+				mean: 891.7830434782592,
+				median: 201.91,
+				minimum: 5.87,
+				mode: 9.15,
+				stdev: 1075.6086847575084,
+				example: 5,
+			},
+		},
+		{
+			name: 'diagnosis',
+			type: 'number',
+			nature: 'discrete',
+		},
+		{
+			name: 'test',
+			type: 'number',
+			nature: 'discrete',
+		},
+		{
+			name: ' letter',
+			type: 'string',
+			nature: 'nominal',
+		},
+		{
+			name: ' letter2',
+			type: 'string',
+			nature: 'nominal',
+		},
+		{
+			name: ' letter3',
+			type: 'string',
+			nature: 'nominal',
+		},
+	],
+} as CodebookSchema
+
+const Template: ComponentStory<typeof CodebookFieldEditor> = ({
+	...args
+}): JSX.Element => {
+	const [field, setField] = useState(codebookResult.fields[0] as Field)
+
+	return <CodebookFieldEditor field={field} onChange={setField} {...args} />
+}
+
+export const Single = Template.bind({})
```

Comment:
If you only have one story and you name the export the same as the parent, Storybook will automatically roll it up to the parent node so you don't have to expand to find the child. In this example, naming const Single as CodebookFieldEditor would result in a single entry in Storybook tree, without the "Single" child you have to expand to.